src/_color-themes.scss

```diff
@@ -0,0 +1,103 @@
+// Themes
+
+$bg: "bg";
+$fg: "fg";
+$lightest-blue: "lightest-blue";
+$lighter-blue: "lighter-blue";
+$light-blue: "light-blue";
+$blue-custom: "blue-custom";
+$teal-custom: "teal-custom";
+$baby-blue: "baby-blue";
+$light-red: "light-red";
+$lighter-red: "lighter-red";
+$shadow-grey: "shadow-grey";
+$white-custom: "white-custom";
+$black-custom: "black-custom";
+$lighter-gray: "lighter-gray";
+$light-gray: "light-gray";
+$dark-gray: "dark-gray";
+$buttonborder-custom: "buttonborder-custom";
+$form-control-border: "form-control-border";
+$input-group-text-border: "input-group-text-border";
+
+$themes: (
+  default: (
+    bg: #fff,
+    fg: gray("800"),
+    lightest-blue: #e7f0ff,
```

Comment:
Are these duplicates of the values in variables.scss? If so, I think we should re-use what is declared there somehow. If not, I am wondering where they came from.

Reply:
Yes they are and I agree. In a similar vein, I think it's better to use variable names like `$main-accent` instead of `$teal` in the themes. The current state is quite confusing.

Reply:
@JosefBud You OK with changing the var names to be semantic rather than literal?

Reply:
@ayan4m1 Sorry I'm a little late here, but I have no problem with that at all!